src/pybind/mgr/cephadm/module.py

```diff
@@ -1311,6 +1307,18 @@ def _check_valid_addr(self, host: str, addr: str) -> str:
 > ssh -F ssh_config -i ~/cephadm_private_key {self.ssh_user}@{addr}'''
             raise OrchestratorError(msg)
 
+        if ipaddress.ip_address(ip_addr).is_loopback and host == addr:
```

Comment:
```suggestion
        if ipaddress.ip_address(ip_addr).is_loopback:
```
do we really want to allow users to explicitly set a loopback addr?

for example: `ceph orch host add node1 127.0.0.1`

Reply:
Well. We have a dedicated flag for single-host clusters. Why disallowing this, if it is clear that this cluster won't extend to another host? I mean, I see your point. WDYT, how can we reduce the pain for users only needing this kind of a cluster?

Reply:
I had a similar worry to @sebastian-philipp that maybe somebody would explicitly want a loopback address for some one node test cluster so I avoided blocking it out entirely while still not assigning a loopback address on our own and creating unexpected issues.